pkg/workflow/constants.go

```diff
@@ -0,0 +1,6 @@
+package workflow
+
+const (
+	// ScanDoneEventSchema is the JSON schema for the scan done event.
```

Comment:
`ScanDoneEventSchema`? This doesn't match the symbol... is there something missing here?

Reply:
Hm… weird ghost in the machine. Removed :broom: